### PR TITLE
Add Exercise01

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: Exercise01/Model/CreateData.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise01.Model
+{
+    class CreateData
```
Comment (voquanghoa) [inline Exercise01/Model/CreateData.cs]: Never name a class starts with a verb. It should be `StudentGenerator` or `StudentFactory`

File: Exercise01/Model/CreateData.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+            for (int i = 1; i <= 10; i++)
+            {
+                list.Add(new Student { Name = $"{(char)(i + 'a' - 1)}", Score = i, Birthday = new DateTime(1109 + i, 11, 1) });
+            }
+        }
+
+        public List<Student> GetList()
+        {
+            return list;
```
Comment (voquanghoa) [inline Exercise01/Model/CreateData.cs]: Remove field, contructor, do everything here and the method name is `CreateStudents`

File: Exercise01/Model/CreateData.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    {
+        private List<Student> list;
+
+        public CreateData()
+        {
+            list = new List<Student>();
+
+            for (int i = 1; i <= 10; i++)
+            {
+                list.Add(new Student { Name = $"{(char)(i + 'a' - 1)}", Score = i, Birthday = new DateTime(1109 + i, 11, 1) });
```
Comment (voquanghoa) [inline Exercise01/Model/CreateData.cs]: When you use `object initializer`, put a statement in a line.

File: Exercise01/Program.cs [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var list = (new CreateData()).GetList();
+            list.ForEach(Console.WriteLine);
+            Console.WriteLine();
+
+            var newList = list.Where(x => x.Score >= 8).OrderBy(x => x.Birthday).ToList();
```
Comment (voquanghoa) [inline Exercise01/Program.cs]: bad naming `newList`. It should be `goodStudents`

File: Exercise01/Program.cs [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+            {
+                newList.ForEach(Console.WriteLine);
+            }
+            else
+            {
+                Console.WriteLine("List is empty");
+            }
+            Console.WriteLine();
+
+            newList = list.Where(x => x.Score >= 5 && x.Score <= 7.5).OrderBy(x => x.Name).ToList();
```
Comment (voquanghoa) [inline Exercise01/Program.cs]: Don't reuse variable if the purpose is changed.

File: Exercise01/Model/CreateData.cs (hunk 1 above)
Comment (huynhngocminh2511) [reply Exercise01/Model/CreateData.cs]: Done

File: Exercise01/Model/CreateData.cs (hunk 3 above)
Comment (huynhngocminh2511) [reply Exercise01/Model/CreateData.cs]: Done

File: Exercise01/Model/CreateData.cs (hunk 2 above)
Comment (huynhngocminh2511) [reply Exercise01/Model/CreateData.cs]: Done

File: Exercise01/Program.cs (hunk 4 above)
Comment (huynhngocminh2511) [reply Exercise01/Program.cs]: Done

File: Exercise01/Program.cs (hunk 5 above)
Comment (huynhngocminh2511) [reply Exercise01/Program.cs]: Done

**Pull request merged**